### PR TITLE
chore: add conditional rendering of "My Lists"

### PR DESCRIPTION
**Pull request by Codycody31**
- When no lists, hide the section in the sidebar

### CODE REVIEW

File: ui/src/scss/_list.scss [hunk 1]
```diff
@@ -113,9 +113,8 @@
     padding-top: 0;
     .page-middle {
         .list-head {
+            margin-top: 15px;
```
Comment (previnder) [inline ui/src/scss/_list.scss:116]: I'm not sure what this is for?

Comment (Codycody31) [reply ui/src/scss/_list.scss:116]: it's mainly just to align the list head with other elements such as the list summary, as the rest of the heads across other pages never touch the header and also go with a similar layout.

Comment (noClaps) [reply ui/src/scss/_list.scss:116]: Actually, some of the other pages also do touch the header, e.g. the profile page.

Comment (Codycody31) [reply ui/src/scss/_list.scss:116]: ah, I'll just yoink that margin out then

Comment (previnder) [reply ui/src/scss/_list.scss]: Thanks.

**Pull request merged**